VideoApi/VideoApi/appsettings.Development.json

```diff
@@ -11,5 +11,5 @@
       }
     }
   },
-  "UseStub": "true"
+  "UseStub": "false"
```

Comment:
revert?

Reply:
yup accidental change to check everything was fine locally